readthedocs/core/static-src/core/js/readthedocs-doc-embed.js

```diff
@@ -19,10 +19,22 @@ var search = require('./doc-embed/search');
         }
     }
 
-    domReady(function () {
+    function init() {
         footer.init();
         sphinx.init();
         search.init();
         sponsorship.init();
-    });
+    }
+
+    // Inject JQuery if isn't present already.
+    if (!window.jQuery) {
```

Comment:
What's up if this line is executed first and then jQuery is loaded by a different extension/theme/etc?

I'm thinking about this scenario;

1. this code is run and we inject jQuery
2. another .js file is loaded that includes jQuery
3. once `domReady` is called, we will have 2 different jQuery versions/files loaded

Shouldn't we inject jQuery synchronously inside `domReady` so we wait for all the scripts to be loaded first and avoid this scenario?